app/recipe/tests.py

```diff
@@ -0,0 +1,109 @@
+import uuid
+from django.urls import reverse
+from django.test import TestCase
+
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from .models import Recipe, Ingredient
+from .serializers import RecipeSerializer
+
+RECIPE_URL = reverse('recipe:recipe-list')
+
+def recipe_detail_url(recipe_id):
+    return reverse('recipe:recipe-detail', args=[recipe_id])
+
+
+class RecipeApiTest(TestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_get_all_recipes(self):
```

Comment:
Maybe `test_list_recipes` is a better name, "list" is a common verb for such an endpoint, not "get all"